templates/zerver/help/add-bots-or-integrations.md

```diff
@@ -0,0 +1,48 @@
+# Add a Bot or Integration
+
+Bots and Integrations are features that are accesible to all members of an Organization.
```

Comment:
I don't think we capitalize `Bot` or `Integration`, unless they're part of a name (e.g. `Your Bots` tab).